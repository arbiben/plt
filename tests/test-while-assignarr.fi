int main(){
int i?
arr int 5 a?
int count?
count = 0?

while(count < 5){
print(a@count)?
count = count + 1?
}

count=0?

while(count < 5){
a[count] = count *3?
count = count + 1?
}

count = 0?

while(count < 5){
print(a@count)?
count = count + 1?
}

i = 5?

while(i>0){
print(i)?
i = i-1?
}
print(i)?
return 0?
}
