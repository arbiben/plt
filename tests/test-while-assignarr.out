00000036912543210
